### PR TITLE
B kiddo patch 3

### PR DESCRIPTION
**Pull request by BKiddo**

### CODE REVIEW

File: 2018/2018-11-13.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+- The forum will be turned on tonight. 
+- The flag mounting will be given to competitors, and strongly recommended in the docs. 
+### Specific
+- Alistair will draw up a budget containing other expenditure and calculate the upper bound of the venue allowance.
+- Kajetan will ask Joyce for help booking SUSU. 
+- Kajetan will update the venue requirement spreadsheet and have a look at other options.
+- Andy BS will look at contacting the Publicity Officer about SUSU.
+- Andy BS will help integrate mentoring timeslots into the rule email announcement.
+- Peter will check the docs and website for any remaining references to team budgets and remove them.
+- Peter will organise a physical meet to discuss kickstart.
```
Comment (PeterJCLaw) [inline 2018/2018-11-13.md]: The intent here is to have an in-person retro for _all_ of the Core Team's efforts so far, not just Kickstart. (Have raised as #62).

File: 2018/2018-11-13.md [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+- The rules need to be shipped. 
+- The forum will be turned on tonight. 
+- The flag mounting will be given to competitors, and strongly recommended in the docs. 
+### Specific
+- Alistair will draw up a budget containing other expenditure and calculate the upper bound of the venue allowance.
+- Kajetan will ask Joyce for help booking SUSU. 
+- Kajetan will update the venue requirement spreadsheet and have a look at other options.
+- Andy BS will look at contacting the Publicity Officer about SUSU.
+- Andy BS will help integrate mentoring timeslots into the rule email announcement.
+- Peter will check the docs and website for any remaining references to team budgets and remove them.
```
Comment (PeterJCLaw) [inline 2018/2018-11-13.md]: Raised as #61.

File: 2018/2018-11-13.md (hunk 1 above)
Comment (BKiddo) [reply 2018/2018-11-13.md]: Ah ok, my bad, edited!

File: 2018/2018-11-13.md [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+- Peter will organise a physical retrospective meet to evaluate core team activities so far, and discuss potential imrpovements in future years.
```
Comment (PeterJCLaw) [inline 2018/2018-11-13.md]: Raised as #62.

**Pull request merged**